docs/Problem-Solving-Examples-in-RxJava.md

```diff
@@ -47,10 +70,23 @@ summer.subscribe({println(it);});
 How could you create an Observable that emits [the Fibonacci sequence](http://en.wikipedia.org/wiki/Fibonacci_number)?
 
 The most direct way would be to use the [`create`](Creating-Observables#wiki-create) operator to make an Observable "from scratch," and then use a traditional loop within the closure you pass to that operator to generate the sequence. Something like this:
+### Java
+```java
+Observable<Integer> fibonacci = Observable.create(observer -> {
```

Comment:
Could you use v2 terminology here? `emitter ->`

Reply:
done! Thank for reviewing 👍